actionmailbox/test/dummy/config/environments/test.rb

```diff
@@ -46,4 +46,7 @@
 
   # Annotate rendered view with file names
   # config.action_view.annotate_rendered_view_with_filenames = true
+  #
+
+  config.action_mailbox.storage_service = :test_email
```

Comment:
I think it would be easier to test this if you removed this line. Then the tests would fall back to the Active Storage service by default. Then in one specific test you could do something like

```ruby
test "you can change the storage service" do
  ActionMailbox.storage_service = :test_email
  # do stuff, basically the test you have below
  ensure
    ActionMailbox.storage_service = nil
end
```

Reply:
Done